reana-ui/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Reana-UI",
```

Comment:
Capital letters REANA here and everywhere else

Reply:
I guess the name of the package still needs to be `reana-ui`, right?

Reply:
yes! you are right https://docs.npmjs.com/files/package.json#name